### ISSUE TITLE
Can we specify the language for generating content, such as French, German, Chinese, etc

### ISSUE THREAD

**Issue by jackqdldd** [e-p-armstrong/augmentoolkit#76]
Restricting the output language in prompt words may not work for some outputs

**Comment by e-p-armstrong** [comment]
Yeah this is often asked for, I'm thinking of making an optional "translation" step after the main pipeline executes to translate the outputs into the desired language... might not be ideal for international INPUT text though. Would the post-execution translation step be a good solution for your usecase?

**Comment by jackqdldd** [comment]
> Yeah this is often asked for, I'm thinking of making an optional "translation" step after the main pipeline executes to translate the outputs into the desired language... might not be ideal for international INPUT text though. Would the post-execution translation step be a good solution for your usecase?

It  should be able to solve the above problem！My current way is to add a restriction on the output language at the end of the prompt like "the generated qa-tuples must be in chinese!"  :D

**Comment by Profxmas** [comment]
> > Yeah this is often asked for, I'm thinking of making an optional "translation" step after the main pipeline executes to translate the outputs into the desired language... might not be ideal for international INPUT text though. Would the post-execution translation step be a good solution for your usecase?
> 
> It should be able to solve the above problem！My current way is to add a restriction on the output language at the end of the prompt like "the generated qa-tuples must be in chinese!" :D

For each prompt ?
